pandas/io/common.py

```diff
@@ -935,7 +930,7 @@ def __init__(
         self.decode = decode
 
         self.attributes = {}
-        for attribute in ("seekable", "readable", "writeable"):
```

Comment:
Should have been `"writable"` but it seems it was never used as we do not support writing to memory-mapped files.